src/sagemaker/jumpstart/exceptions.py

```diff
@@ -169,3 +209,26 @@ def __init__(
             )
 
         super().__init__(self.message)
+
+
+class MarketplaceModelSubscriptionError(ValueError):
+    """Exception raised when trying to deploy a JumpStart Marketplace model.
+
+    A caller is required to subscribe to the Marketplace product in order to deploy.
+    This exception is raised when a caller tries to deploy a JumpStart Marketplace
+    but the caller is not subscribed to the model.
+    """
+
+    def __init__(
+        self,
+        model_subscription_link: Optional[str] = None,
+        message: Optional[str] = None,
+    ):
+        if message:
+            self.message = message
+        else:
+            self.message = "You have not subscribed to this Marketplace model. "
```

Comment:
Replace: To use a proprietary JumpStart model, you must first subscribe to the model in AWS Marketplace.